modules/API/pages/gsql-endpoints.adoc

```diff
@@ -1769,13 +1769,61 @@ Sample Response::
 ====
 
 '''
+
+=== get latest schema version
+
+`GET /gsql/v1/schema/version`
+
+Retrieve the current committed schema version.
+
+==== Permissions
+
+Requires the `READ SCHEMA` privilege.
+
+==== Parameters:
+
+None
+
+==== Example:
+
+[.tabs]
+====
+Sample Request::
++
+--
+[source,bash]
+----
+curl --user tigergraph:tigergraph http://<tigergraph-host>:14240/gsql/v1/schema/version
```

Comment:
The format of curl command example is not consistent with the style of the existing endpoint examples.